GOVERNANCE.md

```diff
@@ -1,11 +1,11 @@
-# io.js Project Governance
```

Comment:
Changes to these files may not be necessary. Refer https://github.com/nodejs/node/pull/2125#issuecomment-119345851